app/controllers/api/v1/rents_controller.rb

```diff
@@ -1,13 +1,15 @@
 module Api
   module V1
     class RentsController < ApiController
+      include Wor::Paginate
       def index
         render_paginated Rent
       end
 
       def create
         rent = Rent.new(rent_params)
         if rent.save
+          UserMailer.new_rent(rent).deliver_later
```

Comment:
Remember that you should not pass objects to a sidekiq job

Reply:
Exactly. 
I'm not pretty sure but this stores elements on another database(redis, which is key-value if I'm not wrong, feel free to correct me).
That database could be anywhere, in the case is in the same machine but in my last project, it was in a separate Amazon Web Service.(Elasticcache). Then sidekiq execute the job and gather the parameter from there. 
So you only should send "plain"(integers, strings, etc) parameters to things like this or async jobs.

Reply:
Mmmm just the id?

Reply:
Ok!!